templates/gsl.install.sh

```diff
@@ -490,44 +548,36 @@ for generate.repository by name as _repository
     documentation(_repository, my.install)
 
     heading1("Define constants.")
-    define_build_directory(_repository)
-    define_icu(my.install)
-    define_zlib(my.install)
-    define_png(my.install)
-    define_qrencode(my.install)
-    define_zmq(my.install)
-    define_mbedtls(my.install)
-    define_boost(my.install)
+    define_build_directory(_repository)    
 
-    heading1("Define utility functions.")
-    define_utility_functions()
-    define_help(_repository, my.install, "install")
+    define_main()
+    define_intialize_the_build_environemnt()
+    define_build(my.install)    
 
     heading1("Initialize the build environment.")
```

Comment:
This comment now applies to defining a group of functions, not performing initialization.